### DISCUSSION TITLE
Artillery Sidewinder X2 Unable to upgrade TFT Controller v1.2. with bootloader V3.0.4

### DISCUSSION THREAD

**Discussion by Houseman74** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2937] [Q&A]
Hi forum. Sorry for annoying question again.
Have X2 with TFT Controller v1.2. with bootloader V3.0.4 (GD32 chipset).
It's been a month now that I am unable to flash new TFT firmware. It simply skips the upgrade. It is not a SD card problem since I successfully get the original FW flash.
I have followed complete readme file, have tried all methods proposed: uppercase, lowercase empty mkstft28.bin with mkstft28_evo.bin renamed, only one file, complete directory proposed by ciotto user (AX2_New_TFT.zip) file. Nothing. Simply nothing.
The update process skips showing Artillery LOGO. Any help would be very appreciated. Attached the screenshot of my actual SD dir content:
<img width="855" alt="Schermata 2024-05-24 alle 23 20 24" src="https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/28305571/3daa3292-41c7-4b4b-aee2-c296bb7b6689">
What am I doing wrong?
Wasn't the upgrade successful after bug solved in January 2024? Best regards and thanks 
Steve

**Comment by kisslorand** [comment]
Check out the instructions [here](https://github.com/kisslorand/BTT-TFT-FW?tab=readme-ov-file#special-notes-for-artillery-mks_gd-tfts), you're missing the "mks_config.txt" file from your SD card.
